libcodechecker/analyze/analysis_manager.py

```diff
@@ -237,13 +237,30 @@ def check(check_data):
             # sequence, whereas clang-tidy does not. We rewrite the file
             # names to contain no escape sequences for every analyzer.
             result_file = rh.analyzer_result_file.replace(r'\ ', ' ')
+            result_base = os.path.basename(result_file)
+            capture_analysis_output = False
+            if hasattr(args, 'capture_analysis_output'):
+                capture_analysis_output = args.capture_analysis_output
 
             if rh.analyzer_returncode == 0:
                 # Analysis was successful processing results.
-                if rh.analyzer_stdout != '':
+                if (capture_analysis_output):
+                    success_dir = os.path.join(output_dir, "success")
+                    if not os.path.exists(success_dir):
+                        os.makedirs(success_dir)
+                if len(rh.analyzer_stdout) > 0:
                     LOG.debug_analyzer('\n' + rh.analyzer_stdout)
-                if rh.analyzer_stderr != '':
+                    if (capture_analysis_output):
+                        with open(os.path.join(success_dir, result_base) +
+                                  ".stdout", 'w') as outf:
```

Comment:
I'd use `.stdout.txt` and `.stderr.txt` (or `.log`!) so that file types are made apparent in a graphical browser.